staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -77,3 +77,27 @@ func (o *RecommendedOptions) ApplyTo(config *server.Config) error {
 
 	return nil
 }
+
+func (o *RecommendedOptions) Validate() []error {
```

Comment:
Since these are the public "RecommendedOptions" for external re-use, can we get a godoc on what this validates or why it should be called?